python-frontend/src/test/java/org/sonar/python/tree/TreeUtilsTest.java

```diff
@@ -90,6 +92,21 @@ public void tokens() {
 
   }
 
+  @Test
+  public void non_whitespace_tokens() {
+    FileInput parsed = parse("if foo:\n  pass");
+    IfStatement ifStmt = (IfStatement) parsed.statements().statements().get(0);
+    List<Token> collect = new ArrayList<>(ifStmt.children().stream().map(t -> t.is(Kind.TOKEN) ? (Token) t : t.firstToken()).collect(Collectors.toList()));
+    collect.add(parsed.lastToken());
+    assertThat(TreeUtils.tokens(parsed)).containsExactly(collect.toArray(new Token[0]));
```

Comment:
I don't see the point of this assertion: how does it help testing `nonWhitespaceTokens`?